lib/katello/tasks/reset.rake

```diff
@@ -29,9 +29,6 @@ namespace :katello do
   task :reset_default_smart_proxy do
     User.current = User.anonymous_admin
     hostname = Socket.gethostname.chomp
-    Setting['ssl_ca_file'] = "/etc/pki/katello/certs/katello-default-ca.crt"
-    Setting['ssl_certificate'] = "/etc/pki/katello/certs/#{hostname}-foreman-client.crt"
-    Setting['ssl_priv_key'] = "/etc/pki/katello/private/#{hostname}-foreman-client.key"
     SmartProxy.where(name: hostname, url: "https://#{hostname}:9090").first_or_create!
```

Comment:
This line will fail if the previous settings are not set.  Removing these lines isn't the proper solution, instead we should figure out why its failing witha  readonly error and fix that

Reply:
It's read only because these settings come from settings.yaml vs. the user Settings table, right?

Reply:
@jlsherrill is katello:reset available and/or actually used in production instances? Just curious if the context is applicable to only our dev environments

Reply:
@akofink no, those should come from the settings table

@johnpmitsch no, only used in devel envs

Reply:
In `foreman/config/settings.yaml` in my devel environment, I have:

```yaml
# SSL settings
:ssl_ca_file: /etc/pki/katello/certs/katello-default-ca.crt
:ssl_certificate: /etc/pki/katello/certs/centos7-devel.akofink-desktop-foreman-client.crt
:ssl_priv_key: /etc/pki/katello/private/centos7-devel.akofink-desktop-foreman-client.key
```

Maybe we're setting it as part of the katello_devel scenario? And if it's set in the config, it can't be changed by the user?